<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(56, 56, 56);</string>
  </property>
  <widget class="QFrame" name="i_Header">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #252424</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="i_closewindow">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>10</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>RB</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 8px;
	background-color: #38DBD0;
	color: #383838
}

QPushButton:hover {
	background-color: #23b2a8;
	color:  #383838
}

QPushButton:pressed {
	background-color: #38DBD0;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="i_minmizewindow">
    <property name="geometry">
     <rect>
      <x>1144</x>
      <y>10</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>RB</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 8px;
	background-color: #38DBD0;
	color: #383838
}

QPushButton:hover {
	background-color: #23b2a8;
	color:  #383838
}

QPushButton:pressed {
	background-color: #38DBD0;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="i_logout">
    <property name="geometry">
     <rect>
      <x>1118</x>
      <y>11</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/logout.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>41</y>
     <width>891</width>
     <height>741</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(38, 38, 38);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="i_Choices">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="i_ViewReportsWid">
     <widget class="QLabel" name="i_TableTitle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-10</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>RB</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #38DBD0;</string>
      </property>
      <property name="text">
       <string>View Reports - Courses</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="i_tablesWidgets">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>871</width>
        <height>691</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="i_CoursesTable">
       <widget class="QTableWidget" name="i_Courses">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>871</width>
          <height>691</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>RB</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QHeaderView::section{background-color:#38DBD0; color:#383838}
QTableWidget::item {
		
	selection-background-color: #38DBD0;
    color: white;
}
QTableWidget::section{

  border-collapse: collapse;
  border-radius: 1em;
  overflow: hidden

}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideLeft</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">New Column</string>
         </property>
         <property name="font">
          <font>
           <family>RB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="foreground">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
         <property name="font">
          <font>
           <family>RB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
         <property name="font">
          <font>
           <family>RB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="i_ClassSelected">
       <widget class="QTableWidget" name="i_Courses_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>871</width>
          <height>691</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>RB</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QHeaderView::section{background-color:#38DBD0; color:#383838}
QTableWidget::item {
		
	selection-background-color: #38DBD0;
  	width: 100%;

}

QTableWidget::item:nth-child(0) {
		

}
QTableWidget::section{

  border-collapse: collapse;
  border-radius: 1em;
  overflow: hidden;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideLeft</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">New Column</string>
         </property>
         <property name="font">
          <font>
           <family>RB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="foreground">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
         <property name="font">
          <font>
           <family>RB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
         <property name="font">
          <font>
           <family>RB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
         <property name="font">
          <font>
           <family>RB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
         <property name="font">
          <font>
           <family>RB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="i_AttendnaceReport">
       <widget class="QTableWidget" name="i_Courses_attendnace">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>871</width>
          <height>641</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>RB</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QHeaderView::section{background-color:#38DBD0; color:#383838}
QTableWidget::item {
		
	selection-background-color: #38DBD0;
    color: white;
  	width: 100%;
}


QTableWidget::section{

  border-collapse: collapse;
  border-radius: 1em;
  overflow: hidden;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideLeft</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
         <property name="font">
          <font>
           <family>RB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
         <property name="font">
          <font>
           <family>RB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
         <property name="font">
          <font>
           <family>RB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="i_save">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>660</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>RB</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-radius: 15px;
	background-color: #38DBD0;
	color:rgb(255, 255, 255);

}

QPushButton:hover {
	background-color: #23b2a8;
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: #38DBD0;
}</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="i_BehaviourReport">
       <widget class="QTableWidget" name="i_Courses_behaviour">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>871</width>
          <height>671</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>RB</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QHeaderView::section{background-color:#38DBD0; color:#383838}
QTableWidget::item {
		
	selection-background-color: #38DBD0;
    color: white;
  	width: 100%;

}
QTableWidget::section{

  border-collapse: collapse;
  border-radius: 1em;
  overflow: hidden;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideLeft</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
         <property name="font">
          <font>
           <family>RB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
         <property name="font">
          <font>
           <family>RB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
         <property name="font">
          <font>
           <family>RB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
         <property name="font">
          <font>
           <family>RB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="i_StartSessionWid">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>RB</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #38DBD0;</string>
      </property>
      <property name="text">
       <string>Start Session</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="i_EndSessionWid">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>RB</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #38DBD0;</string>
      </property>
      <property name="text">
       <string>End Session</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>42</y>
     <width>301</width>
     <height>731</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/Background.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/SAQER1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>787</y>
     <width>1201</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #38DBD0</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="i_ViewReports">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 25px;
	background-color: #38DBD0;
	color:rgb(255, 255, 255);

}

QPushButton:hover {
	background-color: #23b2a8;
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: #38DBD0;
}</string>
   </property>
   <property name="text">
    <string>View Reports</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/reports.png</normaloff>../images/reports.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="i_StartSession">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 25px;
	background-color: #38DBD0;
	color: #ffffff
}

QPushButton:hover {
	background-color: #23b2a8;
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: #38DBD0;
}</string>
   </property>
   <property name="text">
    <string>Start Session</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/play-button.png</normaloff>../images/play-button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="i_EndSession">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 25px;
	background-color: #38DBD0;
	color:#ffffff
}

QPushButton:hover {
	background-color: #23b2a8;
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: #38DBD0;
}</string>
   </property>
   <property name="text">
    <string>End Session</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/stop-button.png</normaloff>../images/stop-button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
